client/test/bpmn-io-modelers/dmn/DmnModelerSpec.js

```diff
@@ -1,446 +1,440 @@
-    modeler._overview.once('views.changed', VERY_LOW_PRIORITY, resolve);
-
-    modeler._overview.on('import.done', ({ err, warnings }) => {
-
-      // assume
-      expect(err).not.to.exist;
-      expect(warnings).to.be.empty;
-    });
-  });
-
-  const modelerImport = new Promise(resolve => {
-    modeler.once('views.changed', VERY_LOW_PRIORITY, resolve);
-
-    modeler.importXML(diagramXML, (err, warnings) => {
-
-      // assume
-      expect(err).not.to.exist;
-      expect(warnings).to.be.empty;
-    });
-  });
-
-  return Promise.all([ overviewImport, modelerImport ]).then(() => modeler);
-}
-
-function inlineCSS(css) {
-  var head = document.head || document.getElementsByTagName('head')[ 0 ],
-      style = document.createElement('style');
-
-  style.type = 'text/css';
-
-  if (style.styleSheet) {
-    style.styleSheet.cssText = css;
-  } else {
-    style.appendChild(document.createTextNode(css));
-  }
-
-  head.appendChild(style);
-}
-
-function openDecisionTable(modeler) {
-  const views = modeler.getViews();
-
-  const view = views.find(({ type }) => type === 'decisionTable');
-
-  return new Promise(resolve => {
-    modeler.once('views.changed', VERY_LOW_PRIORITY, ({ activeView }) => {
-
-      // assume
-      expect(activeView.type).to.equal('decisionTable');
-
-      resolve(activeView);
-    });
-
-    modeler.open(view);
-  });
-}
-
-function openLiteralExpression(modeler) {
-  const views = modeler.getViews();
-
-  const view = views.find(({ type }) => type === 'literalExpression');
-
-  return new Promise(resolve => {
-    modeler.once('views.changed', VERY_LOW_PRIORITY, ({ activeView }) => {
-
-      // assume
-      expect(activeView.type).to.equal('literalExpression');
-
-      resolve(activeView);
-    });
-
-    modeler.open(view);
-  });
-}
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+/* global sinon */
+
+import TestContainer from 'mocha-test-container-support';
+
+import DmnModeler from '../../../src/app/tabs/dmn/modeler/DmnModeler';
+import DrdViewer from '../../../src/app/tabs/dmn/modeler/DrdViewer';
+
+import diagramXML from './diagram.dmn';
+
+const DEFAULT_OPTIONS = {
+  exporter: {
+    name: 'my-tool',
+    version: '120-beta.100'
+  }
+};
+
+const OVERVIEW_ZOOM_SCALE = 0.66;
+
+const VERY_LOW_PRIORITY = 100;
+
+inlineCSS(require('camunda-dmn-js/dist/assets/camunda-platform-modeler.css'));
+
+inlineCSS(`
+  .test-content-container {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .modeler-container,
+  .overview-container {
+    height: 100%;
+  }
+
+  .overview-container {
+    width: 200px;
+  }
+`);
+
+
+describe('DmnModeler', function() {
```

Comment:
What happened there? Full file changed?

Reply:
I think it's automatic CRLF -> LF end of line change according to our [editorconfig](https://github.com/camunda/camunda-modeler/blob/develop/.editorconfig#L6)

Reply:
Note that the file was added by @philippfromme 🪟 _🪟